OsmAnd/res/values/strings.xml

```diff
@@ -13,6 +13,10 @@
     <string name="popular_places_nearby">Popular places nearby</string>
     <string name="recently_visited">Recently visited</string>
 
+    <string name="map_widget_route_information_distance_mode">Distance</string>
```

Comment:
We have several strings with same word, can we reuse something? for example map_widget_trip_recording_distance